includes/im_user_userheader_block.inc

```diff
@@ -584,7 +583,7 @@ if(!empty ($user_fields->field_user_stores['und'][0]['value'])) {
               ->execute();
              
 	    $storeRegion = $fetchRegion->fetchfield();		   
-	    $regionOptions = _im_sales_plan_get_taxonomy_tid();
+	    //$regionOptions = _im_sales_plan_get_taxonomy_tid();

```

Comment:
we have a policy **do not commit commented out code** - if you have an idea or something that you are "in the middle of" or "trying" etc, make a branch for it and keep the production code clean. This is also informed by Kent Beck - "Complete it or DELETE IT"